### PR TITLE
jormungandr: commercial_modes and physical_modes in stop_area

### PR DESCRIPTION
**Pull request by kadhikari**
Ticket: https://jira.canaltp.fr/browse/NAVP-752

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 1]
```diff
@@ -213,6 +218,24 @@ class StopAreaSerializer(serpy.DictSerializer):
     name = NestedPropertyField(attr='properties.geocoding.name', display_none=True)
     administrative_regions = AdministrativeRegionsSerializer()
     timezone = NestedPropertyField(attr='properties.geocoding.timezone')
+    commercial_modes = jsonschema.MethodField(display_none=True)
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: you have a `NestedPropertyField` can you use that one?

Comment (kadhikari) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: In the response bragi is of type dict but not pbfield

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 2]
```diff
@@ -213,6 +218,24 @@ class StopAreaSerializer(serpy.DictSerializer):
...
+    physical_modes = jsonschema.MethodField(display_none=True)
+
+    def fill_modes(self, modes):
+        resp = []
+        if not modes:
+            return resp
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: easier to read: `return []`

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 3]
```diff
@@ -213,6 +218,24 @@ class StopAreaSerializer(serpy.DictSerializer):
...
+        for mode in modes:
+            resp.append(GenericSerializer(mode).data)
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: `return [GenericSerializer(mode).data for mode in modes]`

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 4]
```diff
@@ -213,6 +218,24 @@ class StopAreaSerializer(serpy.DictSerializer):
     name = NestedPropertyField(attr='properties.geocoding.name', display_none=True)
     administrative_regions = AdministrativeRegionsSerializer()
     timezone = NestedPropertyField(attr='properties.geocoding.timezone')
+    commercial_modes = jsonschema.MethodField(display_none=True)
+    physical_modes = jsonschema.MethodField(display_none=True)
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: I don't think that we want to display them if they are None

**Review by kinnou02 — CHANGES_REQUESTED** [review]

**Review by kinnou02 — APPROVED** [review]

**Comment by xlqian** [comment]
Jenkins, test this please!

**Review by xlqian — APPROVED** [review]

**Pull request merged**